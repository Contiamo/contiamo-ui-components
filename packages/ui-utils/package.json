{
  "name": "contiamo-ui-utils",
  "version": "0.0.1-44",
  "description": "A collection of utilities used within the Contiamo UI suite",
  "main": "./lib/index.js",
  "typings": "./lib/index.d.ts",
  "repository": "git@github.com:Contiamo/ui-components.git",
  "author": "Tejas Kumar <tejas@contiamo.com>",
  "license": "MIT",
  "scripts": {
    "package": "rm -rf lib && tsc -d --outDir './lib'",
    "package:watch": "rm -rf lib && tsc -d -w",
    "test": "jest",
    "test:watch": "jest --watch",
    "lint": "tslint './src/**/*.{ts,tsx}' --fix",
    "prettier": "prettier './src/**/*.{ts,tsx}' --write",
    "precommit": "npm run test && npm run lint && npm run prettier && npm run package && git add .",
    "preversion": "npm run precommit",
    "version": "npm run package && git add ."
  },
  "dependencies": {
    "contiamo-ui-theme": "^0.0.1-44",
    "glamor": "^2.20.40",
    "glamorous": "^4.9.7",
    "react": "^16.0.0",
    "tinycolor2": "^1.4.1"
  }
}
