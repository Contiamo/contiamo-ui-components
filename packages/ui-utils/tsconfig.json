{
  "compilerOptions": {
    "outDir": "lib",
    "target": "ES5",
    "module": "commonjs",
    "sourceMap": true,
    "noImplicitAny": true,
    "jsx": "react",
    "lib": ["ES6", "DOM", "DOM.Iterable", "ScriptHost"],
    "moduleResolution": "node"
  },
  "include": ["./src/index.ts", "./types/**/*"],
  "exclude": ["./node_modules"]
}
