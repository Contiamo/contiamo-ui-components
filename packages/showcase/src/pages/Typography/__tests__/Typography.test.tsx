import * as React from "react"
import { render } from "enzyme"

import { wrapTheme } from "@contiamo/ui-utils"
import { contiamoTheme } from "@contiamo/ui"
import ThemelessTypography from "../Typography"

const Typography = wrapTheme(contiamoTheme)(ThemelessTypography)

describe("Tooltips Page", () => {
  it("Should render correctly", () => {
    expect(render(<Typography />)).toMatchSnapshot()
  })
})
