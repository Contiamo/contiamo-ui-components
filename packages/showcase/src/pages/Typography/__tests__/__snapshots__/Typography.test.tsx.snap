// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltips Page Should render correctly 1`] = `
<div
  class="css-1dkvbfz"
>
  <div
    class="css-imab3e"
  >
    Typography
  </div>
  <p>
    <em>
      contiamo-ui-components
    </em>
     keeps typography styles: by default, there are two font sizes and weights, organized in 4 typography styles, as follows:
  </p>
  <div
    class="playground"
  >
    <div
      class="playgroundCode"
    >
      <div
        class="ReactCodeMirror playgroundStage"
      >
        <textarea
          autocomplete="off"
        >
          [object Object]
        </textarea>
      </div>
    </div>
    <div
      class="playgroundPreview"
    >
      <div>
        <div
          class="previewArea"
        />
      </div>
    </div>
  </div>
</div>
`;
